domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -34,13 +37,20 @@ public MeasurementsSyncService(ILogger<MeasurementsSyncService> logger, ISliding
         _measurementSyncMetrics = measurementSyncMetrics;
         _measurementSyncPublisher = measurementSyncPublisher;
         _meteringPointsClient = meteringPointsClient;
+        _options = _measurementSyncOptions;
     }
 
     public async Task FetchAndPublishMeasurements(MeteringPointSyncInfo syncInfo, MeteringPointTimeSeriesSlidingWindow slidingWindow,
         CancellationToken stoppingToken)
     {
         var synchronizationPoint = UnixTimestamp.Now().RoundToLatestHour();
```

Comment:
synchronizationPoint <-- I know you didn't name this. Maybe this should be name synchronizationTimestamp. Point sounds like it's a class / object with fields.

Reply:
Renamed for clarity.

It has been renamed to `pointInTimeItShouldSyncUpTo`